### PR TITLE
New locations

### PR DESCRIPTION
**Pull request by zqtcao**
added: dubai, london, forbidden city and petra

### CODE REVIEW

File: api/cloud/imagesets/dubai.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+            imageUrl: 'http://tabtrekker.parseapp.com/images/dubai/7.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/dubai_villa/7566100368',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/dubai/8.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/roevin/5263878756',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/dubai/9.jpg'
+        }
```
Comment (gowong) [inline api/cloud/imagesets/dubai.js]: Missing comma

File: api/cloud/imagesets/forbidden_city.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            imageUrl: 'http://tabtrekker.parseapp.com/images/forbidden_city/7.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/good_day/1480953347',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/forbidden_city/8.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/mal-b/6591040471',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/forbidden_city/9.jpg'
+        }
```
Comment (gowong) [inline api/cloud/imagesets/forbidden_city.js]: Missing comma

File: api/cloud/imagesets/london.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+            imageUrl: 'http://tabtrekker.parseapp.com/images/london/7.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/jaggers/10545653746',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/london/8.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/christianreimer/14871063270',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/london/9.jpg'
+        }
```
Comment (gowong) [inline api/cloud/imagesets/london.js]: Missing comma

File: api/cloud/imagesets/petra.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+            imageUrl: 'http://tabtrekker.parseapp.com/images/petra/6.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/douglaspperkins/286630893',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/petra/7.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/glouk/6878482358',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/petra/8.jpg'
+        }
```
Comment (gowong) [inline api/cloud/imagesets/petra.js]: Missing comma

**Comment by zqtcao** [comment]
fixed issues, added angkor wat and machu picchu

File: api/cloud/imagesets/angkor_wat.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/miguelitosql/5951112293',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/angkor_wat/7.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/97238650@N08/9041891490',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/angkor_wat/8.jpg'
+        }
+
```
Comment (gowong) [inline api/cloud/imagesets/angkor_wat.js]: Remove the extra new line

File: api/cloud/imagesets/angkor_wat.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+            infoUrl: 'https://www.flickr.com/photos/9084427@N07/16558631219',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/angkor_wat/4.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/aschaf/11902304005',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/angkor_wat/5.jpg'
+        },
+        {
+            infoUrl: 'https://www.flickr.com/photos/image-catalog/19146403798',
+            imageUrl: 'http://tabtrekker.parseapp.com/images/angkor_wat/5.jpg'
```
Comment (gowong) [inline api/cloud/imagesets/angkor_wat.js]: There's two images named "5.jpg"

File: LOCATIONS.md [hunk 7]
```diff
@@ -46,13 +46,14 @@
...
+- Petra, Jordan ***(Done)***
 - Chichen Itza, Mexico ***(Done)***
 - Parthenon, Greece
-- Machu Picchu, Peru
+- Machu Picchu, Peru ***(Done)***
+- Angkor Wat, Cambodia ***(Done)***
 - Taj Mahal, India
 - St. Basil's Cathedral, Moscow
-- Forbidden City, Beijing, China
+- Forbidden City, Beijing, China ***(Done)***
```
Comment (gowong) [inline LOCATIONS.md]: Change this to "Forbidden City, China"

File: api/cloud/imagesets/forbidden_city.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+exports.ImageSet = {
+    name: 'Forbidden City, Beijing, China',
```
Comment (gowong) [inline api/cloud/imagesets/forbidden_city.js]: Change this to "Forbidden City, China"

**Pull request closed without merging**